### ISSUE TITLE
The documentation in this repo seems really out of date

### ISSUE THREAD

**Issue by farringtonp** [k8-proxy/cs-k8s-api#486]
The documentation seems out of date and misleading for this repo. It needs to be updated. 

Too many references to ICAP which undermines the true utility of this API for all use cases

**Comment by GINAGC** [comment]
Yes, we have been updating only the wiki, we will take care of this.

**Comment by Lestat06** [comment]
@GINAGC are we talking about readme.md or documentation.md? either way these documentation and the steps there need to be also viewd from someone from the devs since there are new steps that are being used now for the deployment and i will need assistance with that

**Comment by GINAGC** [comment]
@Lestat06 lets merge those MD files in readme. Please update the sections that you know. Add here the sections where you need support (after your changes) and I will ask dev to support. 

We have the most recent information on wiki, so you can take documentation from there as needed.

**Comment by Lestat06** [comment]
@GINAGC please review  https://github.com/k8-proxy/cs-k8s-api/pull/571
also we should probably remove the other versions from the table and leave only golang-ck8s or what do u suggest
also let me know when u confirm to delete **documentation.md** **limitations.md** **ApiEndpointsDocumentation.md**

**Comment by GINAGC** [comment]
@Lestat06 merged and some feedback:

- please remove documentation.md
- remove the table of versions, we don't need that anymore. 
- add a list of endpoints to the readme
- update the architecture diagram to be the one with minio. I will point you this diagram. 

Thank you

**Comment by Lestat06** [comment]
ok great thanks, will update once u point the diagram

**Comment by Lestat06** [comment]
@GINAGC all the above are updated, all that is left is for some dev to check the deplpoyment since im pretty sure it has changed and to check the limitiations
